src/creator.html

```diff
@@ -169,7 +170,26 @@ <h3>Phrase</h3>
 			<span>{{currentPage+1}}/{{numberOfPages()}}</span>
 			<button ng-disabled="currentPage >= numberOfPages() - 1" ng-click="currentPage=currentPage+1">Next</button>
 		</div>
-
+		<div ng-show= "questionBankDialog" class="overlay"></div>
+		<div class="questionBank-wrapper" ng-show= "questionBankDialog">
+				<div class="questionBank-content">
+					<strong>Enable question bank? </strong>
+					<br/>
+					<span>
+						<input type="radio" ng-model="enableQuestionBank" ng-value=false> No</input>
+						<input type="radio" ng-model="enableQuestionBank" ng-value=true> Yes</input>
+					</span>
+					<br/><br/>
```

Comment:
This is something we can fix with a styling pass that's due anyhow, but as a general rule, avoid using `<br />`. It's an HTML element that serves no purpose beyond styling, which is the purview of CSS.